fxcore/chain.json

```diff
@@ -137,15 +137,95 @@
           "type": "cosmos",
           "version": "0.46"
         }
+      },
+      {
+        "name": "v5.0.x",
+        "tag": "v5.0.0",
+        "height": 11601700,
+        "proposal": 41,
+        "recommended_version": "v5.0.0",
+        "compatible_versions": [
+          "v5.0.0"
+        ],
+        "binaries": {
+          "darwin/arm64": "https://github.com/functionX/fx-core/releases/download/v5.0.0/fx-core_Darwin_arm64.tar.gz",
+          "darwin/amd64": "https://github.com/functionX/fx-core/releases/download/v5.0.0/fx-core_Darwin_amd64.tar.gz",
+          "linux/arm64": "https://github.com/functionX/fx-core/releases/download/v5.0.0/fx-core_Linux_arm64.tar.gz",
+          "linux/amd64": "https://github.com/functionX/fx-core/releases/download/v5.0.0/fx-core_Linux_amd64.tar.gz",
+          "windows/x86_64": "https://github.com/functionX/fx-core/releases/download/v5.0.0/fx-core_Windows_x86_64.zip"
```

Comment:
checks are failing because this property name isn't recognized in the chain Schema file. There exists windows/arm64 and windows/amd64, but no other windows/ properties.

Reply:
@JeremyParish69 OK, fixed, thank you.